lib/static/modules/reducer.js

```diff
@@ -75,7 +80,8 @@ export default function reducer(state = getInitialState(compiledData), action) {
                     apiValues,
                     view: {scaleImages, lazyLoadOffset}
                 },
-                formattedSuites
+                formattedSuites,
+                {view}
```

Comment:
Did **this** line help to solve the problem with `scale images` & `lazy load` buttons?